packages/opensearch-client/package.json

```
@@ -22,7 +22,6 @@
         "build:watch": "yarn build --watch",
         "test": "TEST_ELASTICSEARCH='true' node ../scripts/bin/ts-scripts test ../opensearch-client --",
         "test:debug": "TEST_ELASTICSEARCH='true' node ../scripts/bin/ts-scripts test --debug ../opensearch-client --",
-        "test:elasticsearch6": "TEST_ELASTICSEARCH='true' ELASTICSEARCH_VERSION='6.8.6' node ../scripts/bin/ts-scripts test ../opensearch-client --",
         "test:elasticsearch7": "TEST_ELASTICSEARCH='true' ELASTICSEARCH_VERSION='7.9.3' node ../scripts/bin/ts-scripts test ../opensearch-client --",
         "test:elasticsearch8": "TEST_ELASTICSEARCH='true' ELASTICSEARCH_VERSION='8.1.2' node ../scripts/bin/ts-scripts test ../opensearch-client --",
         "test:opensearch1": "TEST_OPENSEARCH='true' node ../scripts/bin/ts-scripts test ../opensearch-client --",
```

```suggestion
        "test:opensearch1": "TEST_OPENSEARCH='true' OPENSEARCH_VERSION='1.3.11' node ../scripts/bin/ts-scripts test ../opensearch-client --",
```
OPENSEARCH_VERSION can be removed from OS2 if we want